spec/features/confirm_membership_invitation_spec.rb

```diff
@@ -1,26 +0,0 @@
-describe 'Confirming an invitation to a new organisation', type: :feature do
-  let(:organisation) { create(:organisation) }
```

Comment:
Duplicate of `'Confirming an invite to a second or subsequent organisation'` below